File: example/javabuilds/5-test-suite/bar/test/src/bar/BarTests.java
package bar;

import static org.junit.Assert.assertTrue;
import org.junit.Test;

public class BarTests {

  @Test
  public void hello() {
    String result = Bar.hello();
    assertTrue(result.startsWith("Hello"));
  }

  @Test
  public void world() {
    String result = Bar.hello();
    assertTrue(result.endsWith("World"));
  }
}

File: example/scalalib/testing/1-test-suite/bar/test/src/BarTests.scala
package bar
import utest.*
object BarTests extends TestSuite {
  def tests = Tests {
    test("hello") {
      val result = Bar.hello()
      assert(result.startsWith("Hello"))
      result
    }
    test("world") {
      val result = Bar.hello()
      assert(result.endsWith("World"))
      result
    }
  }
}
